### DISCUSSION TITLE
instalação com espd-dmx cores

### DISCUSSION THREAD

**Discussion by VARIAVEIS-LAB** [someweisguy/esp_dmx#72] [General]
I'm trying to use your esp_dmx to make a project, I just need the red one, but I'm not getting it, can you help me?

**Comment by VARIAVEIS-LAB** [comment]
me parece que mesmo deixando so um canal, sempre reage de forma aleatoria

**Reply by someweisguy** [reply]
Hi! Thanks for reaching out. I can try to help, but I will need a bit more information. Firstly, which version of this library are you using, and are you using Arduino or ESP-IDF? Are you able to post your code?
